dist-git/copr_dist_git/package_import.py

```diff
@@ -218,6 +224,15 @@ def import_package(opts, namespace, branches, srpm_path, pkg_name):
         try:
             if not branch_commits:
                 upload_files = commands.import_srpm(srpm_path)
+                # in case of importing, the content of directory in `reponame`
+                # changes. To update the state of `Commands` class isn't an easy
+                # process - look how much logic pyrpkg.cli.cliClient.load_cmd has
+                # and the logic is written for context in cliClient, not for Commands
+                # which we are using. Considering this wouldn't be an easy task for
+                # pyrpkg to implement, we can refresh the Commands class after every
+                # import - even if it is not elegant
+                # note: if https://pagure.io/rpkg/issue/690 is resolved, you may delete this
+                commands = _load_command(opts, reponame, repo_dir)
```

Comment:
@onosek do you think this change makes sense?

Reply:
Ping @onosek :-) seems like we could rather fix the library.  Can you give us guidance?

Reply:
I prepared a fix that potentially solves the issue:
https://bodhi.fedoraproject.org/updates/?search=rpkg-1.66-8